nutils/evaluable.py

```diff
@@ -4116,6 +4116,9 @@ def _derivative(self, var, seen):
     def evalf(f, sh1, sh2):
         return f.reshape(f.shape[:-1] + (sh1, sh2))
 
+    def _compile_expression(self, py_self, add_constant, f, sh1, sh2):
+        return _pyast.Variable('evaluable').get_attr('Unravel').get_attr('evalf').call(f, sh1, sh2)
```

Comment:
The commit message said Ravel, should be Unravel.

Reply:
Fixed.